geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/JdbcConnectorException.java

```diff
@@ -25,7 +29,51 @@
 public class JdbcConnectorException extends CacheRuntimeException {
```

Comment:
Mentioning use of getMessage() than getCause(), in absence of drivers in the classpath; may help users.